examples/DGmethods/ex_001_dycoms.jl

```diff
@@ -115,14 +118,62 @@ function Initialise_DYCOMS!(state::Vars, aux::Vars, (x,y,z), t)
   state.moisture.ρq_tot = ρ * q_tot
 end   
 
+# TODO: temporary; move to new CLIMA module
+function gather_diags(dg, Q)
```

Comment:
```suggestion
function gather_diagnostics(dg, Q)
```
(`diag` is also used in Julia for the diagonal elements of an array)